package/gluon-debug/files/lib/gluon/respondd/proc.cache

```diff
@@ -0,0 +1 @@
+10000
```

Comment:
what do we need this file for?

Reply:
It tells respondd how long to cache the query result.

Reply:
The package provides the `proc` response, so it only makes sense that it would decide how often new data is relevant.